isat/pkg/storage/local/local.py

```diff
@@ -0,0 +1,13 @@
+import os
+from isat.pkg.models.image import Image
+
+
+class Local_Storage:
```

Comment:
Bad naming. Please, do not use _ in classnames in Python (and in any other language, actually).